docs/src/prototypes/popups/GridImagePicker/GridImagePickerItem.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react'
+
+import { Image, Button } from '@stardust-ui/react'
+
+import _ from 'lodash'
```

Comment:
Too many empty lines 😸 

```suggestion
import { Image, Button } from '@stardust-ui/react'
import _ from 'lodash'
import * as React from 'react'
```

Reply:
I put label ```wip``` to this PR hours ago, and removed ```ready for review``` 😄 
Sorry for the confusion, working on it after comments from @jurokapsiar. Anyway, thank you for the initial code review